projects/vdk-plugins/vdk-ipython/src/vdk_ipython.py

```diff
@@ -7,28 +7,55 @@
 from IPython.core.magic_arguments import argument
 from IPython.core.magic_arguments import magic_arguments
 from IPython.core.magic_arguments import parse_argstring
-from vdk.internal.builtin_plugins.run.standalone_data_job import (
-    StandaloneDataJobFactory,
-)
+from vdk.internal.builtin_plugins.run import standalone_data_job
+
+
+class JobControl:
+    """
+    The main idea of thi class is to make it easier to  work with IStandaloneDataJob instances
+    Since the user will be using this class
+    """
```

Comment:
Trying to figure out why it would be easier to work with a wrapper than the VDK class?
Didn't get that info from the PR description, nor the docs

Reply:
I thing, it will looks strange to use data_job.__enter__ (if I was a standard user I would not think it return an initialised job_input). Instead having a class that is more user friendly is a better option in my opinion. I am not sure whether my class is that good, but I think I will be nice to have something like a "wrapper" for the users. I am open to any other ideas